noether_conversions/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ find_package(catkin REQUIRED COMPONENTS
   visualization_msgs
 )
 
+find_package(tool_path_planner REQUIRED)
```

Comment:
Shouldn't there be a dependency added on the `noether::tool_path_planner` target as well?